### PR TITLE
Feature/create zaak zac

### PR DESCRIPTION
**Pull request by damm89**

### CODE REVIEW

File: backend/src/zac/core/api/views.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def get_object(self):
 
 
 class CreateZaakView(views.APIView):
-    authentication_classes = (authentication.SessionAuthentication,)
+    # authentication_classes = (authentication.SessionAuthentication,)
```
Comment (damm89) [inline backend/src/zac/core/api/views.py:227]:
```suggestion
    authentication_classes = (authentication.SessionAuthentication,)
```

**Review by svenvandescheur — CHANGES_REQUESTED** [review]
Few remarks

File: frontend/zac-ui/libs/features/zaak-detail/src/lib/actions/keten-processen/configuration-components/start-process/role-step/role-step.component.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<div class="start-process__role-step pt-4" *ngIf="startProcessRoleForm">
+  <ng-container [formGroup]="startProcessRoleForm">
+    <div class="row" formArrayName="roles" *ngFor="let role of rolesControl.controls; let i = index">
+      <!-- Free user input  -->
+      <gu-input
```
[the remark this answers is not in the document]
Comment (kelvincy) [reply frontend/zac-ui/libs/features/zaak-detail/src/lib/actions/keten-processen/configuration-components/start-process/role-step/role-step.component.html:5]: yes, still used

File: frontend/zac-ui/libs/features/start-case/src/lib/features-start-case.component.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        this.form = this.getForm();
+      }, (error) => console.error(error),
+    );
+  }
+
+  /**
+   * Returns the field configurations for the form.
+   * @param {Zaak} zaak If given, preselect zaak.
+   */
+  getForm(zaak: Zaak = null): FieldConfiguration[] {
```
[the remark this answers is not in the document]
Comment (kelvincy) [reply frontend/zac-ui/libs/features/start-case/src/lib/features-start-case.component.ts:61]: yes, still used

**Pull request merged**